### ISSUE TITLE
Can this be dropped in place of tilelive-bridge

### ISSUE THREAD

**Issue by pnorman** [mojodna/tilelive-tmsource#12]
For the WMF map stack we need to deal with tm2source styles, but the code currently requires Mapnik XML for tilelive-bridge. I'm [looking at how to best do this](https://phabricator.wikimedia.org/T192058) and was wondering if tilelive-tmsource is pretty much compatible with tilelive-bridge in terms of features, except reading tm2source YML instead of XML

**Comment by mojodna** [comment]
Yes. It wraps `tilelive-bridge` internally and converts the YAML to Mapnik XML before handing it over (via an `xml` property on the URI object).
